runtime/near-vm-runner/fuzz/fuzz_targets/runner.rs

```diff
@@ -86,18 +88,67 @@ fn create_context(input: Vec<u8>) -> VMContext {
     }
 }
 
+/// Define a configuration for which [`available_imports`] is implemented. This
+/// allows to specify the imports available in a [`ConfiguredModule`].
+///
+/// [`available_imports`]: wasm_smith::Config::available_imports
+/// [`ConfiguredModule`]: wasm_smith::ConfiguredModule
+#[derive(Arbitrary, Debug)]
+struct ModuleConfig {}
+
+impl wasm_smith::Config for ModuleConfig {
+    /// Returns a WebAssembly module which imports all near host functions. The
+    /// imports are grabbed from a compiled [test contract] which calls every
+    /// host function in its method `sanity_check`.
+    ///
+    /// [test contract]: near_test_contracts::rs_contract
+    fn available_imports(&self) -> Option<std::borrow::Cow<'_, [u8]>> {
+        static MODULE: OnceCell<Vec<u8>> = OnceCell::new();
+        Some(MODULE.get_or_init(|| filter_module_imports(rs_contract())).into())
```

Comment:
I believe it shouldn't be necessary to filter out anything from the example contract at all – we could just supply the contract as is and `wasm-smith` would pick out the imports by itself.

Though I do think that the assertion which verifies there are any imports being specified at all in `filter_module_imports` is useful.